docs/architecture/sstable/sstable3/index.rst

```diff
@@ -12,6 +12,7 @@ ScyllaDB SSTable - 3.x
 
 .. include:: ../_common/sstable_what_is.rst
 
+* In ScyllaDB 6.0 and above, the ``me`` format is mandatory, and ``md`` format is used only when uprading from an existing cluster using ``md``. The ``sstable_format`` parameter is ignored, if it is set to ``md``.
```

Comment:
nit: s/uprading/upgrading/
nit: s/ignored,/ignored/ (comma is not needed)

ditto 6.0 vs. 5.5